crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -1287,7 +1289,7 @@ impl OlmMachine {
             }
             AnyDecryptedOlmEvent::RoomKeyBundle(e) => {
                 debug!("Received a room key bundle event {:?}", e);
```

Comment:
pre-existing, but is it fine to log a full key bundle event? (no keys will appear in there?)